src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -56,6 +59,11 @@ class JvmCompile(CompilerOptionSetsMixin, NailgunTaskBase):
 
   size_estimators = create_size_estimators()
 
+  class Compiler(enum.Enum):
```

Comment:
We're not using this in the option value for the compiler.

Reply:
i think i missed this comment, will circle back

Reply:
IMO, can just merge on green and then delete this later.